UnitystationLauncher/ViewModels/ChangelogViewModel.cs

```diff
@@ -1,41 +1,52 @@
+using System.Collections.Generic;
 using ReactiveUI;
 using Octokit;
 using System.Collections.ObjectModel;
 using System.Reactive.Concurrency;
+using System.Threading;
 using System.Threading.Tasks;
+using Serilog;
 
 namespace UnitystationLauncher.ViewModels
 {
     public class ChangelogViewModel : ViewModelBase
     {
-        readonly GitHubClient _client;
-        ObservableCollection<PullRequestViewModel> _pullRequests = new ObservableCollection<PullRequestViewModel>();
+        private readonly GitHubClient _client;
 
-        public ObservableCollection<PullRequestViewModel> PullRequests
-        {
-            get => _pullRequests;
-            set => this.RaiseAndSetIfChanged(ref _pullRequests, value);
-        }
+        ObservableCollection<PullRequestViewModel> PullRequests { get; }
 
         public ChangelogViewModel()
         {
             _client = new GitHubClient(new ProductHeaderValue("UnitystationCommitNews"));
-            RxApp.MainThreadScheduler.ScheduleAsync((scheduler, ct) => GetPullRequestsAsync());
+            PullRequests = new();
+
+            RxApp.TaskpoolScheduler.ScheduleAsync(GetPullRequestsAsync);
         }
 
-        public async Task GetPullRequestsAsync()
+        private async Task GetPullRequestsAsync(IScheduler scheduler, CancellationToken cancellationToken)
         {
```

Comment:
Codacy says to remove this, but this is part of the required method signature by the task scheduler. Even if we aren't using these parameters we have to have them.

Reply:
![image](https://user-images.githubusercontent.com/5573038/222845669-28beb6a2-221e-49ce-9602-87e790449cf8.png)

Removing this:
![image](https://user-images.githubusercontent.com/5573038/222845893-ad66ac9b-4a34-428e-93ba-ab82b55b90e5.png)